functions/utils/utils.js

```diff
@@ -194,6 +194,21 @@ function filterOutKtx(libaries) {
   return libaries.filter((library) => !library.includes("/ktx"));
 }
 
+/**
+ * Gets the stack trace from an error.
+ *
+ * @param {Error} error - The error to get the stack trace from.
+ * @throws {Error} If the provided argument is not an Error object.
+ * @return {string} The stack trace.
+ */
+function getStackTrace(error) {
+  if (!(error instanceof Error)) {
+    throw new Error("Provided argument is not an Error object");
```

Comment:
Either log or return the argument in the message here btw, so folks have an idea of what was incorrectly passed.